pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -1211,6 +1211,23 @@ let
         };
       };
 
+      gencay.vscode-chatgpt = buildVscodeMarketplaceExtension {
+        meta = with lib; {
```

```suggestion
        meta = {
```
As a rule of thumb, avoid `with`, specially nested with, more reasoning:
https://github.com/atorres1985-contrib/nixpkgs/commit/451dfa6d84586b5cb83e6a7254689dff66b4523a

---

Oooh interesting.
Going to remove it.